src/pages/index.js

```diff
@@ -53,18 +58,18 @@ IndexPage.propTypes = {
 
 export const pageQuery = graphql`
   query IndexQuery {
-    allStrapiProject {
+    allStrapiProject (limit: 6){
```

Comment:
Any reason for choosing 6? Why not 4 or 8?

Reply:
I chose 6 because they all fit perfectly on my standard sized browser window and the thumbnails are a slightly smaller size. It's a simple change that I figured would develop more as we went. Would you like me to change it to 4 or 8?

Reply:
Fair answer. Can we come up with a reason that's "deeper"? The off-putting part about 6 is when you scale the window you sometimes get 4 in row 1 and 2 in row 2.

Kickstarter does 4 per row, which is why I asked about 4 or 8...

For now, at a minimum, we need to fix the overlap problems when scaling the window; I'm fine with deciding how many tiles to display later...

Reply:
P.S. Weren't there supposed to be "blurbs" for each project in addition to the image and the name?